### ISSUE TITLE
How to unzip the samples in "dataset/images/pathches.zip"

### ISSUE THREAD

**Issue by Kumalight** [pereira-gha/activefire#2]
size of the downloaded pathches.zip is only 1 kb. When I try to unzip this file by 'python unzip_patches.py',  the following error will appear: 'File is not a zip file'.

**Comment by Minoro** [comment]
Hi, could you try to download the zip files directly from the GitHub repository and replace your local files?
[patches.zip](https://github.com/pereira-gha/activefire/blob/main/dataset/images/patches.zip)
[Or the direct link](https://github.com/pereira-gha/activefire/raw/main/dataset/images/patches.zip)

The patches.zip should be placed at `<your-local-code>/dataset/images/patches.zip`

[patches_goli.zip](https://github.com/pereira-gha/activefire/blob/main/dataset/masks/patches_goli.zip)
[Direct download](https://github.com/pereira-gha/activefire/raw/main/dataset/masks/patches_goli.zip)

[patches_murphy.zip](https://github.com/pereira-gha/activefire/blob/main/dataset/masks/patches_murphy.zip)
[Direct Download](https://github.com/pereira-gha/activefire/raw/main/dataset/masks/patches_murphy.zip)

[patches_schoreder.zip](https://github.com/pereira-gha/activefire/blob/main/dataset/masks/patches_schroeder.zip)
[Direct Download](https://github.com/pereira-gha/activefire/raw/main/dataset/masks/patches_schroeder.zip)

This zip files should be placed at  `<your-local-code>/dataset/patches/patches_*.zip`

Please, let me know if it works for you.

**Comment by Kumalight** [comment]
@Minoro  It works. Thank you very much.

**Comment by Minoro** [comment]
Nice! I'll update the Readme file with this information.

**Comment by pistori** [comment]
Hi, I've tried to download the files directly from the links but it is not working. I've also tried "git lfs pull" but GIT is returning the following message. Any hint or alternative way to download the file? 

$ batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.
$ error: failed to fetch some objects from 'https://github.com/pereira-gha/activefire.git/info/lfs'
